.github/workflows/cont_integration.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           cargo update -p tokio-util --precise "0.7.11"
           cargo update -p indexmap --precise "2.5.0"
           cargo update -p security-framework-sys --precise "2.11.1"
+          cargo update -p rustls --precise "0.21.12"
```

Comment:
Will this work?
```suggestion
          cargo update -p rustls@0.23.18 --precise "0.23.17"
```

Reply:
Yes, it looks like this did the trick. Thanks for the clarification!